pkg/kubelet/cm/cgroup_manager_linux_test.go

```diff
@@ -87,14 +91,48 @@ func TestLibcontainerAdapterAdaptToSystemdAsCgroupFs(t *testing.T) {
 			input:    "/Burstable/pod_123",
 			expected: "Burstable.slice/Burstable-pod_123.slice/",
 		},
+		{
```

Comment:
remove duplicated entry

Reply:
The case test the cgroupName without underscore.

Reply:
when do we have a case without a fully qualified input path?